### PR TITLE
[Merged by Bors] - fix: Fixed instructions to run CLI smoke test in DEVELOPER.md

### PR DESCRIPTION
**Pull request by ozgrakkurt**
Added required commands to run before running smoke tests

### CODE REVIEW

File: DEVELOPER.md [hunk 1]
```diff
@@ -477,6 +477,8 @@ make smoke-test-k8
 Perform CLI smoke test against your running cluster (Kubernetes or local)
 
 ```
+export FLUVIO=<path to fluvio binary,e.g. target/debug/fluvio>
```
Comment (sehz) [inline DEVELOPER.md:480]: prefer explicit working code instead of comment in the code block

**Review by sehz — CHANGES_REQUESTED** [review]

File: DEVELOPER.md (hunk 1 above)
Comment (ozgrakkurt) [reply DEVELOPER.md]: changed it to working code

**Review by sehz — APPROVED** [review]
LGTM.  thanks

**Comment by ozgrakkurt** [comment]
@sehz sorry last second change. I cloned repo from scratch and found out that only one command is needed

**Review by sehz — APPROVED** [review]
LGTM

**Comment by sehz** [comment]
bors retry

**Pull request closed without merging**